client/signup/steps/site-style/style.scss

```diff
@@ -116,7 +116,11 @@
 
 	input[type='radio'] {
 		position: absolute;
-		left: -1000px;
+		left: -10000px;
+		top: auto;
```

Comment:
`top` / `width` and `height` styling here don't seem necessary here. Do I overlook something or we could remove them?

Reply:
`top` probably not. `width` and `height` I added for good measure, you know, for those `10000` pixel screens :trollface: 

I'll update it in the upcoming PR: https://github.com/Automattic/wp-calypso/pull/32165